sql/mysql/driver.go

```diff
@@ -191,6 +191,14 @@ func (d *conn) supportsRenameColumn() bool {
 	return d.gteV(v)
 }
 
+// supportsIndexComment reports if the connected database
+// supports comments on indexes.
+func (d *conn) supportsIndexComment() bool {
+	// According to Oracle release notes, comments on
+	// indexes were added in version 5.5.3.
+	return d.mariadb() || d.gteV("5.5.3")
+}
```

Comment:
```suggestion
func (c *conn) supportsIndexComment() bool {
	// According to Oracle release notes, comments on
	// indexes were added in version 5.5.3.
	return c.mariadb() || c.gteV("5.5.3")
}
```

lint fails as master version was changed with different receiver names.